src/Math-Complex/PMComplex.class.st

```diff
@@ -543,8 +543,10 @@ PMComplex >> isComplexConjugateOf: aNumber [
 
 { #category : #testing }
 PMComplex >> isComplexConjugateOfAComplexNumber: aComplexNumber [
+
 	"Answer true if self and aComplexNumber are complex conjugates of each other. The complex conjugate of a complex number is the number with an equal real part and an imaginary part equal in magnitude but opposite in sign."
-	^ (aComplexNumber real = real) and: [ aComplexNumber imaginary = (-1 * imaginary) ]
+
+	^ self = aComplexNumber complexConjugate
```

Comment:
Does this need to a method? It's only used by a Matrix to check it is Hermitian. I don't feel it's needed. Is there evidence to the contrary?

Reply:
Issue #247